pkg/jx/cmd/create_mlquickstart.go

```diff
@@ -120,12 +120,12 @@ func NewCmdCreateMLQuickstart(commonOpts *opts.CommonOptions) *cobra.Command {
 
 // Run implements the generic Create command
 func (o *CreateMLQuickstartOptions) Run() error {
-	o.Debugf("Running CreateMLQuickstart...\n")
+	log.Logger().Debugf("Running CreateMLQuickstart...\n")
 
 	interactive := true
 	if o.BatchMode {
 		interactive = false
-		o.Debugf("In batch mode.\n")
+		log.Logger().Debugf("In batch mode.\n")
```

Comment:
+1 for replacing o.Debug calls with a call to log.Logger()

Reply:
I also need to get ride of the \n at the end of every statement, but that can happen in another PR

Reply:
Well, you might as well do it now ;-)